lib/NumericKeys.js

```diff
@@ -3,49 +3,72 @@
 Object.defineProperty(exports, "__esModule", {
   value: true
 });
-exports.default = void 0;
+exports["default"] = void 0;
 
 var _react = _interopRequireDefault(require("react"));
 
+var _propTypes = _interopRequireDefault(require("prop-types"));
+
 var _KeyButton = _interopRequireDefault(require("./KeyButton"));
 
-function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+var _classnames = _interopRequireDefault(require("classnames"));
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }
+
+var numericKeysClassMap = {
+  '.': 'num-key-left',
+  CE: 'num-key-right',
+  0: 'num-key-zero',
+  1: 'num-key-one',
+  2: 'num-key-two',
+  3: 'num-key-three',
+  4: 'num-key-four',
+  5: 'num-key-five',
+  6: 'num-key-six',
+  7: 'num-key-seven',
+  8: 'num-key-eight',
+  9: 'num-key-nine'
+};
+var numericKeysOrder = [1, 2, 3, 4, 5, 6, 7, 8, 9, '.', 0, 'CE'];
```

Comment:
These individually exposed components are problematic in that they include items in global space that should be well wrapped-up.